sass/lightbox.scss

```diff
@@ -78,7 +79,6 @@ html.lightbox-enabled {
   .lightbox-contents {
     @extend %user-select-none;
 
-    cursor: zoom-out;
```

Comment:
dont we want the zoom-out cursor for other browsers? I believe we use it on discover project popup